google/cloud/bigquery/job/base.py

```diff
@@ -722,6 +722,14 @@ def cancelled(self):
             and self.error_result.get("reason") == _STOPPED_REASON
         )
 
+    def __repr__(self):
+        result = (
+            f"{self.__class__.__name__}<"
+            f"project={self.project}, location={self.location}, id={self.job_id}"
+            ">"
```

Comment:
Note - I chose chevrons over parentheses so that the repr string does not appear as something `eval()`-able, as it's often the case with `repr()` output.

Reply:
Makes sense, thanks.